layouts/community/ortho_4x12/guidoism/readme.md

```diff
@@ -1,37 +1,56 @@
 # Guido's Planck Keyboard Layout
 
+I've written a script to convert the json file from the
+[QMK Configurator](https://config.qmk.fm) to the C macros
+and a pretty form the comments. 
+
+First import the json file in this directory into the configurator,
+make any changes, export the file and then run this:
+
+    mv ~/Downloads/guidoism.json layouts/community/ortho_4x12/guidoism/ && python3 layouts/community/ortho_4x12/guidoism/generate_c.py && make planck:guidoism:dfu
+
+TODO: Move running of generate_c.py to makefile so I just need to run `make planck:guidoism:dfu`
```

Comment:
I think you can use the `rules.mk` to do this, actually.  

I'm not entirely sure how, but, if you check the chibos.mk and avr.mk files in tmk_core, that should give you an idea on how.

Reply:
Thank you for the suggestion, I'll take a look at this when I have time.